components/nimbus/src/targeting.rs

```diff
@@ -131,7 +124,27 @@ pub fn jexl_eval<Context: serde::Serialize>(
         })
         .with_transform("bucketSample", bucket_sample);
 
-    let res = evaluator.eval_in_context(expression_statement, context)?;
+    match evaluator.eval_in_context(expression_statement, context) {
+        Ok(v) => Ok(v),
+        Err(err) => Err(NimbusError::EvaluationError(err.to_string())),
```

Comment:
is err a &str? if so, into()

Reply:
err is a `jexl_eval::error::EvaluationError<'a>`, which doesn't support `.into()` on String/&str, but does have a `to_string` method